config/packages/neusta_converter.yaml

```diff
@@ -0,0 +1,25 @@
+neusta_converter:
+  converter:
+    test.person.converter:
+#      converter: Neusta\ConverterBundle\Converter\GenericConverter
+      target_factory: Neusta\ConverterBundle\Tests\Fixtures\Model\PersonFactory
+      populators:
+        - Neusta\ConverterBundle\Tests\Fixtures\Populator\PersonNamePopulator
+#      properties:
```

Comment:
Sollen diese auskommentierten Codezeilen drinnen bleiben?

Reply:
Ja, ich glaube die dienen als "Vorlage" sozusagen.

Reply:
Okay, Sounds like a case for docs for me :)
But nevermind, this is okay for me.